src/mscorlib/src/System/IO/StreamReader.cs

```diff
@@ -723,7 +760,7 @@ public override String ReadLine()
                         String s;
                         if (sb != null) {
                             sb.Append(charBuffer, charPos, i - charPos);
-                            s = sb.ToString();
+                            s = this.GetStringAndReleaseSharedStringBuilder(sb);
```

Comment:
`this.` not needed on method calls either (compare to the call to AcquireSharedStringBuilder)